aries_cloudagent/wallet/askar.py

```diff
@@ -192,8 +192,14 @@ async def get_signing_key(self, verkey: str) -> KeyInfo:
         if not key:
             raise WalletNotFoundError("Unknown key: {}".format(verkey))
         metadata = json.loads(key.metadata or "{}")
+
+        # Provisioned keys don't get tags by default
+        try:
+            kid = key.tags["kid"] if "kid" in key.tags else None
```

Comment:
You can do `key.tags.get('kid')` instead of this. The default if it doesn't exist is `None`.

Reply:
thanks, I prefer this solution which I've just commited

Reply:
actually this might not fix it as the problem is that tags doesn't exist. See my second comment.